dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonUtils_JSON.java

```diff
@@ -174,25 +174,21 @@ private static String jsonifyFFDC(int maxFieldLength, String wlpUserDir,
         StringBuilder sb = CollectorJsonHelpers.startFFDCJsonFields(hostName, wlpUserDir, serverName);
 
         String datetime = CollectorJsonHelpers.dateFormatTL.get().format(ffdcData.getDatetime());
-        CollectorJsonHelpers.addToJSON(sb, FFDCData.getDatetimeKeyJSON(), datetime, FFDCData.getDatetimeOmitBoolJSON(), false, true, false, !(sb.length() > 1), false);
-        CollectorJsonHelpers.addToJSON(sb, FFDCData.getMessageKeyJSON(), ffdcData.getMessage(), FFDCData.getMessageOmitBoolJSON(), false, true, false, !(sb.length() > 1), false);
-        CollectorJsonHelpers.addToJSON(sb, FFDCData.getClassNameKeyJSON(), ffdcData.getClassName(), FFDCData.getClassNameOmitBoolJSON(), false, true, false, !(sb.length() > 1),
-                                       false);
-        CollectorJsonHelpers.addToJSON(sb, FFDCData.getExceptionNameKeyJSON(), ffdcData.getExceptionName(), FFDCData.getExceptionNameOmitBoolJSON(), false, true, false,
-                                       !(sb.length() > 1),
-                                       false);
-        CollectorJsonHelpers.addToJSON(sb, FFDCData.getProbeIdKeyJSON(), ffdcData.getProbeId(), FFDCData.getProbeIdOmitBoolJSON(), false, true, false, !(sb.length() > 1), false);
-        CollectorJsonHelpers.addToJSON(sb, FFDCData.getThreadIdKeyJSON(), DataFormatHelper.padHexString((int) ffdcData.getThreadId(), 8), FFDCData.getThreadIdOmitBoolJSON(), false,
-                                       true, false, !(sb.length() > 1),
-                                       false);
+        CollectorJsonHelpers.addToJSON(sb, FFDCData.getDatetimeKeyJSON(), datetime, false, true, false, !(sb.length() > 1), false);
```

Comment:
addToJSON returns true if it adds anything to sb.  Should use that instead of the more expensive check of sb.length()